src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java

```diff
@@ -1,4 +1,107 @@
 package seedu.linkedout.logic.commands;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.linkedout.logic.commands.CommandTestUtil.VALID_SKILL_JAVA;
+import static seedu.linkedout.logic.commands.CommandTestUtil.VALID_SKILL_MARKETING;
+import static seedu.linkedout.logic.commands.CommandTestUtil.VALID_SKILL_PYTHON;
+import static seedu.linkedout.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.linkedout.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.linkedout.testutil.TypicalApplicants.getTypicalLinkedout;
+import static seedu.linkedout.testutil.TypicalIndexes.INDEX_FIRST_APPLICANT;
+import static seedu.linkedout.testutil.TypicalIndexes.INDEX_SECOND_APPLICANT;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.linkedout.commons.core.Messages;
+import seedu.linkedout.commons.core.index.Index;
+import seedu.linkedout.model.Linkedout;
+import seedu.linkedout.model.Model;
+import seedu.linkedout.model.ModelManager;
+import seedu.linkedout.model.UserPrefs;
+import seedu.linkedout.model.applicant.Applicant;
+import seedu.linkedout.testutil.ApplicantBuilder;
+import seedu.linkedout.testutil.ApplicantUtil;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code AddSkillCommand}
+ */
 public class AddSkillCommandTest {
+
+    private Model model = new ModelManager(getTypicalLinkedout(), new UserPrefs());
+
+    @Test
+    public void execute_addExistingSkills_success() {
+        Applicant firstApplicant = model.getDefaultApplicantList().get(0);
```

Comment:
Can consider using same way of getting applicants as `EditCommandTest` to reduce confusion.
<img width="861" alt="Screenshot 2022-04-05 at 5 47 19 PM" src="https://user-images.githubusercontent.com/77470406/161727348-78eebfcd-c4ef-4a31-a6ee-74e3f8aed76a.png">

Reply:
Thanks for the suggestion. I have fixed both instances.